packages/context/package.json

```diff
@@ -210,14 +210,14 @@
   },
   "author": "Google LLC",
   "dependencies": {
-    "@lit/reactive-element": "^1.6.2 || ^2.0.0"
+    "@lit/reactive-element": "^2.1.0"
```

Comment:
This doesn't look right. I think it would require a breaking change to `@lit/context` to drop `@lit/reactive-element@1.6.x` support. I wonder why the release process updated this this time?

Reply:
good catch. changesets auto adjusts package dependencies for internal packages when the dependent gets a minor bump.
i can manually adjust this before merging, if we want to keep the support for old `@lit/reactive-element`. though i'm not 100% sure if typescript will be happy when using the context decorators on old reactive-element versions.